bin/pycbc_live

```diff
@@ -241,15 +241,20 @@ class LiveEventManager(object):
 
             live_ifos = [ifo for ifo in fud if 'snr_series' in fud[ifo]]
 
+            time_offset = 0
+            rtoff = 'foreground/{}/time_offset'.format(coinc_ifos[0])
+            if rtoff in coinc_results:
+                time_offset = coinc_results[rtoff]
             event = SingleCoincForGraceDB(live_ifos, coinc_results, bank=bank,
                                           psds=psds, followup_data=fud,
                                           low_frequency_cutoff=f_low,
                                           channel_names=args.channel_name,
-                                          mc_area_args=self.mc_area_args)
+                                          mc_area_args=self.mc_area_args,
+                                          time_offset=time_offset)
```

Comment:
Given that the above change is repeated twice (for coincs and singles) would it make sense to move all of that into the constructor of `SingleCoincForGraceDB`?